src/Nethermind/Nethermind.TxPool/TxPoolInfoProvider.cs

```diff
@@ -79,5 +81,48 @@ public TxPoolInfo GetInfo(BlockHeader head)
 
             return new TxPoolInfo(pendingTransactions, queuedTransactions);
         }
+
+        public string GetSnapshot(BlockHeader head)
```

Comment:
Could we return object instead of building string?

Reply:
Yes, but in form of string our return is ready to import to calc sheet. When returning objects, they are separated by planty of brackets, making importing process longer - need to remove them first